tests/integration/conftest.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import pytest
+
+DIR_PATH = os.path.dirname(__file__)
+
+
+def pytest_addoption(parser):
+    parser.addoption("--flannel-version", nargs="?", type=str, default="amd64",
+                     choices=["amd64", "arm64", "s390x"],
+                     help="The version of flannel resource. [amd64/arm64/s390x]")
+    parser.addoption("--flannel-resource", nargs="?", type=str,
+                     default=os.path.join(DIR_PATH, "../../flannel-amd64.tar.gz"),
```

Comment:
I'm not a big fan of these needing to be passed in, rather than having the test code just build the resource in Python, but the build script would need some refactoring to make that reasonable, so it's fine for now.  I would, however, prefer that the default be dropped for `--flannel-resource` because the error that you get if you just run `tox -e integration` is a bit confusing.